scripts/varsome_api_run.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
```

Comment:
Well if you are on a system where python2 is still the default. which is none except Macs, and dont create a python3 virtual env then yes it will fail. 
So leave it on pyton3. The setup.py script has a requirement for python3 but I guess its ok to change that to python3